pycheribuild/projects/cross/cheribsd.py

```diff
@@ -396,12 +403,21 @@ def arch_build_flags(self):
             result = {"TARGET": "riscv", "TARGET_ARCH": target_arch}
         elif self.crosscompile_target.is_i386():
             result = {"TARGET": "i386", "TARGET_ARCH": "i386"}
-        elif self.crosscompile_target.is_aarch64():
+        elif self.crosscompile_target.is_aarch64(include_purecap=True):
             result = {"TARGET": "arm64", "TARGET_ARCH": "aarch64"}
+            if self.crosscompile_target.is_cheri_purecap():
+                result["TARGET_ARCH"] = "morello"  # TODO: aarch64c?
+            if self.crosscompile_target.is_hybrid_or_purecap_cheri():
+                # FIXME: still needed?
+                result["WITH_CHERI"] = True
```

Comment:
@brettferdosi @zxombie Are these still needed?

Reply:
I have `WITH_CHERI` but not `WITH_CHERI128` in my build script.